content/docs/user-guide/experiment-management/sharing.md

```diff
@@ -0,0 +1,283 @@
+```
+
+It pulls all the text files from Git repository and DVC tracked files from DVC
+remote. You need to have both of these remote configured in your project. See
+[above](#prepare-remotes-to-share-experiments) for information regarding remote
+configuration.
+
+When you don't have a default DVC remote, or would like to ask DVC to use a
+particular remote, you can specify it with `--remote` / `-r` option.
+
+DVC can use more than one process to pull DVC tracked files from remotes. You
+can set the number of processes to pull by `--jobs` / `-j` option. By default
+DVC uses `4 * cpu_count()` processes for non-SSH remotes, and `4` processes for
+SSH remotes.
+
+If there is already an experiment in the current repository with the name you're
+trying to pull, DVC won't overwrite it unless you supply `--force` flag.
+
+## Pulling all experiments from a remote
+
+Assuming all experiments have distinct names, you can create a loop to pull all
+experiments from `origin` like the following.
+
+```dvc
+dvc exp list --all --names-only | while read -r expname ; do
+    dvc exp pull origin ${expname}
+done
+```
+
+## Creating a separate directory for an experiment
+
+A very common use case for experiments is to create a separate local directory
+for your work. You can do so by `dvc exp apply` and `dvc exp branch` commands,
+but here we'll see how to use `dvc exp pull` to copy an experiment.
+
+Suppose there is project in `~/my-project` that you have many experiments and
+would like to have a copy of a particular experiment named `exp-abc12` in this
+project.
+
+You first clone the repository to another directory:
+
+```dvc
+$ git clone ~/my-project ~/my-successful-experiment
+$ cd ~/my-successful-experiment
+```
+
+Git sets `origin` of cloned repository to `~/my-project`, so when you list the
+experiments in this new clone, you can see your all experiments in
+`~/my-project`.
+
+```dvc
+$ dvc exp list origin
+main:
+   ...
+   exp-abc12
+   ...
+```
+
+If there is no DVC remote in the original repository, and there is no means to
+set up one, you can define the original repository's DVC cache as a _remote_ in
+the clone.
+
+```dvc
+$ dvc remote add --local --default storage ~/my-project/.dvc/cache
+```
+
+If there is a DVC remote for the project, assuming all DVC cache in
+`~/my-project` repository is pushed to it, you can pull an experiment in the
+clone:
+
+```dvc
+$ dvc exp pull origin exp-abc12
+```
+
+Then we can apply this experiment and get a workspace that contains all your
+experiment files:
+
+```dvc
+$ dvc exp apply exp-abc12
+```
+
+Now you have a dedicated directory for your experiment that contains all your
+artifacts.
+
+---
+
+> BELOW is from GS:Experiments and will be removed
+
+## Sharing Experiment
+
+After committing the best experiments to our Git branch, we can
+[store and share](/doc/start/data-and-model-versioning#storing-and-sharing) them
+remotely like any other iteration of the pipeline.
+
+```dvc
+dvc push
```

Comment:
same here, start with $

Reply:
I'll remove this piece, it's from the current GS:Experiments. Kept for review.